### ISSUE TITLE
Label Inputs

### ISSUE THREAD

**Issue by jpdevries** [robj00/quiz-app#1]
We'll work on this in the call, all inputs should be labeled for accessibility and usability (you should be able to click the label to select the input also)

![](http://j4p.us/141y0d433Y0c/Screen%20Shot%202017-05-17%20at%206.48.12%20PM.png)
